src/utils/download.ts

```diff
@@ -0,0 +1,25 @@
+import * as fs from 'fs';
+import path from 'path';
+import axios, { AxiosResponse } from 'axios';
+
+/**
+ * Downloads file from remote HTTP[S] host and puts its contents to the
+ * specified location.
+ * @param url URL of the file to download.
+ * @param destinationPath Path to the destination file.
```

Comment:
I like this param name better, but it's `filePath` in your function (which I think is ambiguous).